### PR TITLE
Add Tab/Shift+Tab key handlers to editbox

### PR DESCRIPTION
**Pull request by zotho**
Also `editbox` now returns `true` if its state has changed

### CODE REVIEW

File: src/widgets/editbox.rs [hunk 1]
```diff
@@ -314,7 +343,7 @@ impl<'a> Editbox<'a> {
         let pos = context
             .window
             .cursor
-            .fit(size, Layout::Free(Vector2::new(5., 5.)));
+            .fit(size, Layout::Free((5., 5.).into()));
```
Comment (not-fl3) [inline src/widgets/editbox.rs]: I like Vector2::new() a bit more than .into() :) 

And I think that this is outside of the scope of this PR anyway.

Comment (zotho) [reply src/widgets/editbox.rs]: I revert this changes

**Comment by not-fl3** [comment]
Nice, thanks!
